app/templates/main.html

```diff
@@ -1,7 +1,18 @@
 <section>
 	<div>
+		<form>
+			<label>Add a Room!<br>
+			<input type="text" name="add_room" ng-model="roomName" placeholder="Room Name"></label>
+			<input type="button" value="Add Room!" ng-click="main.addRoom(roomName)">
+		</form>
 		<ul ng-repeat="room in rooms">
 			<li>{{room.$value}}</li>
+			<!-- An idea I want to do is have the name of the new room bind instantly to a new dom element, only to destroy itself when user submits form - so it appears to the user their room is being named in real time, even though it's only instantialized upon form submission. Far fetched? -->
```

Comment:
I feel this is poor UI. A room isn't created until they submit the form and I don't see the upside to echoing that as they type. But if you're going to do it you will need a lot of logic to stop having the _fake_ one disappear as the _real_ one is added.